src/math/MathUtils.js

```diff
@@ -54,6 +54,14 @@ const MathUtils = {
 
 	},
 
+	// https://www.gamedev.net/tutorials/programming/general-and-gameplay-programming/inverse-lerp-a-super-useful-yet-often-overlooked-function-r5230/
+
+	inverseLerp: function ( a, b, value ) {
+
+		return MathUtils.clamp( ( value - a ) / ( b - a ) );
```

Comment:
This returns `NaN` right now since `clamp()` requires three parameters. It should be:
```js
return THREE.MathUtils.clamp( ( value - a ) / ( b - a ), 0, 1 );
```